### PR TITLE
Initial import of modules for LG ESS 1.0 VI

### PR DESCRIPTION
**Pull request by sa0win**
Betaversion der Module für Bezug, Speicher und Wechselrichter (PV) für die LG ESS 1.0 VI (erste Generation). 
Die Modulkonfiguration funktioniert, jedoch noch nicht die Langzeitloggings.

PV, Speicher und Bezug können live ausgelesen werden.

### CODE REVIEW

File: modules/bezug_lgessv1/JSON-Beispiele.txt [hunk 1]
```diff
@@ -0,0 +1,69 @@
+Login:
+
+$ curl -k --connect-timeout 5 -d '{"password":"DE1807BKRD0093E8"}' -H "Content-Type: application/json" -X PUT https://192.168.0.248/v1/login
```
Comment (dj3mu) [inline modules/bezug_lgessv1/JSON-Beispiele.txt]: **Attention:**  
Is this your active password that has been published to the world audience now?  
Then remember that it's time to change it!

**Comment by sa0win** [comment]
it is not the current password ... but thanks for having an eye on it.
I'll remove the tempfile ... thanks

**Comment by dj3mu** [comment]
Regarding commit bb229a9c38313592fefbb42f544df135f4a6b021:

Not sure if it was intention or just missed.  
But the "password" is present also in the published `openwb.conf` changes. Maybe that addition needs some reasonable default anyway.

I also noted during implementation for my PR #253 that `openwb.conf` seems to be extended / maintained by `runs\atreboot.sh`.  
I think that is for getting the lines added during updates which must never overwrite `openwb.conf` itself.  
So maybe you want to extend `runs\atreboot.sh`, to add your config lines to `openwb.conf`.

**Comment by snaptec** [comment]
yes!
Please at the variables to runs/atreboot.sh with reasonable demo content.
rest looks good and will be merged.

**Comment by sa0win** [comment]
The password ist also registration number of the PCS as default. So, it makes no sense to prefill it by default with my regnum ;-) 
Nevertheless I know that I have to change it in openwb.conf but time ist the limiting factor this weekend.

Thanks for the hint  with atreboot.sh.

**Pull request merged**